lib/nrf91_lwm2m_app/lwm2m/lwm2m_firmware.c

```diff
@@ -16,15 +16,16 @@
 #include <modem/nrf_modem_lib.h>
 #include <power/reboot.h>
 
-#include "settings.h"
+#include <net/nrf91_lwm2m_app.h>
+#include <net/nrf91_lwm2m_fota.h>
 
 #include <logging/log.h>
-LOG_MODULE_REGISTER(app_lwm2m_firmware, CONFIG_APP_LOG_LEVEL);
+LOG_MODULE_REGISTER(lwm2m_firmware, CONFIG_NRF91_LWM2M_APP_LOG_LEVEL);
 
 #define BYTE_PROGRESS_STEP (1024 * 10)
 #define REBOOT_DELAY       K_SECONDS(1)
 
-#if defined(CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT)
+#if defined(CONFIG_NRF91_LWM2M_APP_FIRMWARE_UPDATE_OBJ_SUPPORT)
```

Comment:
I don't think we need those ifdefs in the source code, this whole file is included conditionally based on `CONFIG_NRF91_LWM2M_APP_FIRMWARE_UPDATE_OBJ_SUPPORT`. A bit weird they were here in the first place, probably some leftover code.

Reply:
Fixed.